orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -3968,6 +3968,47 @@ Return true if all elements are true and false otherwise.
       .SetContextDependentFunctionBodyBuilder(BuildNllLossInternalFunction<13>)
       .TypeAndShapeInferenceFunction([](InferenceContext& ctx) { propagateElemTypeFromInputToOutput(ctx, 0, 0); })
       .SetDoc(R"DOC(NegativeLogLikelihoodLossInternal)DOC");
+
+  ONNX_CONTRIB_OPERATOR_SCHEMA(FakeQuant)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .SetDoc(
+          "FakeQuant operator that fuses quantization->dequantization pattern into a single node. "
+          "FakeQuant takes in a non quantized tensor as input and generates a non quantized tensor as output. "
+          "But internally, it will perform Quantization->Dequantization operation that simulates the effects of "
+          "quantization within the model. Loss in numerical precision introduced by model quantization is "
+          "corrected by adjusting the model weights through the FakeQuant op.")
+      .Input(0, "input_tensor", "Tensor to be fake quantized.", "T")
+      .Input(1, "scale", "Quantization scale.", "T")
+      .Input(2, "zero_point", "Quantization zero point as non quantized type.", "T")
+      .Output(0, "fake_quantized_tensor", "Fake quantized input tensor.", "T")
+      .Output(1, "quantization_mask",
+              "Mask where values indicate if the quantized value was in qmin, qmax range."
+              " Needed for gradient computation.",
+              "T_BOOL")
+      .Attr(
+          "quant_min",
```

Comment:
I'll leave `quant_min` and `quant_max` as is since they indicate the min and max range of the quantized type.
Will update the others to just be `input`, `output` and `mask`.